ios/Surveys/Sources/Presentation/Modules/Home/HomeView.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  HomeView.swift
+//  Surveys
+//
+//  Created by Su Ho on 03/11/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import SwiftUI
+
+struct HomeView: View {
+
+    var body: some View {
+        VStack {
+            HomeHeaderView(
+                title: "Monday, JUNE 16",
+                imageURLString: "https://cdn.pixabay.com/photo/2016/11/18/23/38/child-1837375__340.png"
+            )
+            .padding(EdgeInsets(top: 0.0, leading: 20.0, bottom: 0.0, trailing: 20.0))
+            Spacer()
+        }
+        .background(Color.black) // Remove this later
```

Comment:
even this, `Remove this later` should be a TODO comment to keep track of code refactoring in the end.

If we can not find a good solution yet, please try to use FIXME comment 🙏

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/97/commits/a862c5aa40b0e0180933c63487a93236ff8ba51f